### DISCUSSION TITLE
When is the final pekko 1.1.0 expected to be released

### DISCUSSION THREAD

**Discussion by dasormeter** [apache/pekko#1414] [Q&A]
What is the current expected timeline to release final 1.1.0 of pekko?

We are anxiously awaiting this as it contains a critical fix needed for pekko-grpc-plugin that is a blocker for us

https://github.com/apache/pekko-grpc/pull/222/files/277bed410226acb235f228b7bf361a95967b331e#r1495750001

**Comment by pjfanning** [comment]
We discuss releases on the dev@pekko.apache.org mailing list.

Archive: https://lists.apache.org/list.html?dev@pekko.apache.org

Latest discussion: https://lists.apache.org/thread/6dbj4zpkzyrsdnxpj77643nhlcq8y7wp

Right now, I think we are at least a month from a Pekko core 1.1.0 release. Then possibly another month before a Pekko gRPC release (which would have to be proceeded with a Pekko HTTP release).

I see https://github.com/apache/pekko-grpc/pull/222 as an enhancement as opposed to a bug fix. Why can't you use Scala 2 compatible sources?

**Reply by pjfanning** [reply]
it is up to you but 1.1.0-M1 has not received any negative feedback and may be safe to use if you test carefully before going to production.

**Reply by dasormeter** [reply]
yes we have tested M1 carefully and found it to resolve our blocker and run stably

**Reply by dasormeter** [reply]
> I see [apache/pekko-grpc#222](https://github.com/apache/pekko-grpc/pull/222) as an enhancement as opposed to a bug fix. Why can't you use Scala 2 compatible sources?

It is actually this specific fix that we need (happens to be part of that same PR) -- avoid the null authority assert
https://github.com/apache/pekko-grpc/pull/222/files#r1495750001

**Comment by ramazanyich** [comment]
I see the 1.1.0-RC1 tag, but can not download the dependecies from maven repo. 
got error:
`warn] 	Note: Unresolved dependencies path:
[error] sbt.librarymanagement.ResolveException: Error downloading org.apache.pekko:pekko-actor_2.13:1.1.0-RC1
[error]   Not found
[error]   Not found
[error]   not found: /home/xxx/.ivy2/local/org.apache.pekko/pekko-actor_2.13/1.1.0-RC1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/apache/pekko/pekko-actor_2.13/1.1.0-RC1/pekko-actor_2.13-1.1.0-RC1.pom
[error] Error downloading org.apache.pekko:pekko-stream_2.13:1.1.0-RC1
[error]   Not found
[error]   Not found
[error]   not found: /home/xxx/.ivy2/local/org.apache.pekko/pekko-stream_2.13/1.1.0-RC1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/apache/pekko/pekko-stream_2.13/1.1.0-RC1/pekko-stream_2.13-1.1.0-RC1.pom
[error] 	at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:346)
[error] 	at lmcoursier.CoursierDependencyResolution.$anonfun$update$39(CoursierDependencyResolution.scala:315)
[error] 	at scala.util.Either$LeftProjection.map(Either.scala:573)
[error] 	at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:315)
[error] 	at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
`

**Reply by pjfanning** [reply]
Releases are discussed on the dev@pekko.apache.org mailing list as I described above. This is an ASF requirement.

The vote thread includes instructions on how to set up sbt to find RC jars on repository.apache.org.
https://lists.apache.org/thread/ddbt8vzfvs6ct39xj3y1gksnm5w4b9d2

**Comment by raboof** [comment]
1.1.0 has been released but contains a critical bug (#1463) if you're using sharding. 1.1.1 has a fix and is in the process of being released.
